utils/config/daos_server.yml

```diff
@@ -193,6 +193,17 @@
 ##nr_hugepages: -1
 #
 #
+## Number of CART contexts per secondary provider per engine
+#
+## Specifies the number of secondary CART contexts each engine will create for
+## each additional fabric provider after the first. This only applies when
+## running in multi-provider mode.
+#
+## default: 0
```

Comment:
@frostedcmos I'm wondering, is a secondary context always required if there is a secondary provider? If so maybe the default should be 1, and anything less should be invalid?